docs/integrations/creating-nodes/build/n8n-node.md

```diff
@@ -0,0 +1,186 @@
+---
+#https://www.notion.so/n8n/Frontmatter-432c2b8dff1f43d4b1c8d20075510fe4
+contentType: howto
+search:
+  boost: 3
+---
+
+# Using the n8n-node tool
+
+The `n8n-node` tool is the official CLI for developing community nodes for n8n. You can use it to scaffold out new nodes, build your projects, and run your node as you develop it.
+
+Using `n8n-node`, you can create nodes that adhere to the 
+
+## Prerequisites
+
+Before you can use the `n8n-node` tool, you must [install n8n locally with `npm`](/hosting/installation/npm.md). The `n8n-node` tool runs the local n8n instance with your community node loaded to provide live previews.
```

Comment:
This is no longer needed, I now run `n8n` with `npx` in the CLI so it will auto-install

Reply:
```
npm create @n8n/node@latest

cd n8n-nodes-example
npm run dev # and other commands, using the CLI as a local dependency
```

`npm run dev` will run `npx n8n` as a subprocess, so there is no prerequisite to install n8n beforehand anymore

Reply:
Okay, got it. I initially thought you were saying that you need to explicitly run `npx n8n`, but it sounds like that isn't necessary. Thanks, that'll make this much simpler.